tests/test_integration.py

```diff
@@ -632,7 +699,7 @@ def tearDown(self):
             vm['hostname'] = RenameTest._get_renamed_hostname(vm['hostname'])
 
         clean_cert(vm)
-        clean_all(vm['hostname'])
+        clean_all(self.datacenter_type, vm['hostname'])
```

Comment:
hm i wonder how this is even supposed to work without the `self.route_network`. but it was like that before so i guess we can ignore this for now

Reply:
It was broken before. It used `vm['hostname']` as filter for the route_network to query all Hypervisors and did not find any so did nothing. This might actually be a spot where we are not properly cleaning up things so lets fix it.

Reply:
fixed